src/cljc/rems/text.cljc

```diff
@@ -82,11 +81,13 @@
   "Return the tempura translation for a given key. Additional fallback
   keys can be given."
   [& ks]
-  #?(:clj (tr (conj (vec ks) (text-format :t/missing (vec ks))))
+  #?(:clj (or (tr ks) ; optimization: fetch missing translation only when needed
+              (text-format :t/missing ks))
```

Comment:
this change affects `text` function, not `text-no-fallback`. github diff looks slightly confusing